src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -80,5 +81,28 @@ public async Task<ActionResult> CreateUserAsync(Guid organisationId, CreateUserR
 
             return Ok();
         }
+
+        [Route("users/{userId}")]
```

Comment:
If we're following the convention used for the organizations routes, this probably ought to be `[Route("api/v1/users/{userId}")]`.

Reply:
As an aside, I think we probably ought to have a think about our routing and whether all our action methods currently reside in the most appropriate controllers. For example, the current routing implies that two of the methods in this class probably ought to live in the organizations controller.

I don't think we need to do anything now as part of this change – this comment is more of a note as I think it's probably something that might be worthwhile discussing.